app/mailers/user_mailer.rb

```diff
@@ -13,8 +13,9 @@ def confirmation(user, confirmation_token)
     mail to: @user.confirmable_email, subject: "Confirmation Instructions"
   end
 
-  def password_reset(user)
+  def password_reset(user, password_reset_token)
     @user = user
+    @password_reset_token = password_reset_token
```

Comment:
I'm not 100% sure, but this might need to be escaped using `CGI.escape`. Signed IDs can contain `=` which are not URL safe, but I don't know if they can be safely used in resource params.

Reply:
@ayushn21 thank you for your diligence here. I just tested this locally with a signed_id that contained a `=` and everything worked as expected. The [API](https://api.rubyonrails.org/classes/ActiveRecord/SignedId.html#method-i-signed_id) mentions that "...it's safe to send in an email...", so I _think_ we need to do any additional escaping 🤔

Reply:
Great, if it works it works :). I wasn't sure off the top of my head. It's safe to send in an email but I wondered if it was safe to use in a URL. But if you've tested it and it worked then all good :)